docs/faq.md

```diff
@@ -0,0 +1,33 @@
+# General FAQ
+
+## AMD GPUs crashing the entire system
+
+Changing resolution, using DRI_PRIME and doing various other things can cause crashes, but running the command `echo high| sudo tee /sys/bus/pci/drivers/amdgpu/0000:??:??.?/power_dpm_force_performance_level` or adding the `amdgpu.dpm=0` to the kernel commandline stops these crashes.
+
+## The MacBook's screen freezes, fans run up, and then it turns off
+
+A broken runtime call crashed Linux; try a patched kernel (or updating if you're already using it), or boot macOS and check the crashlog. If you have `apple-bce`, try updating it to 0.2. You may need a custom live ISO. You can also try adding the ``efi=noruntime`` option to GRUB's command line paramaters, or adding ``efivarfs /sys/firmware/efi/efivars efivarfs ro,remount,nofail 0 0`` to the end of your /etc/fstab file (although don't use both at the same time).
+
+## The WiFi GUI is not working
+
+Follow the [wifi guide](https://wiki.t2linux.org/guides/wifi). If you followed that, try updating, or using a different ISO. Some users reported that patched kernel `5.15.12` worked better. You could also try reloading brcmfmac: ``sudo modprobe -r brcmfmac && sudo modprobe brcmfmac``.
```

Comment:
@AdityaGarg8 said the wifi gui [issue was fixed](https://github.com/t2linux/wiki/pull/221#discussion_r790140656) so I don't think we need a section on it, as updating would be enough to fix the issue.